collections/nemo_nlp/nemo_nlp/data/data_layers.py

```diff
@@ -733,6 +735,86 @@ def __init__(self,
         super().__init__(dataset_type, dataset_params, **kwargs)
 
 
+class BertQuestionAnsweringDataLayer(TextDataLayer):
```

Comment:
Please add to your docstring what each of the arguments for the initializer means.